execution-engine/engine-tests/src/test/system_contracts/genesis.rs

```diff
@@ -201,6 +254,7 @@ fn should_fail_if_bad_pos_install_contract_is_provided() {
             pos_installer_bytes,
```

Comment:
@jhyeom26 

execution-engine/engine-tests/src/test/system_contracts/genesis.rs
Will you leave this as it is?

Reply:
No, your correction is right.

Reply:
?? Does it mean to change?